lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -698,3 +698,71 @@ def test_key_summary(self, query_as):
                 "origin": None,
             },
         }
+
+    def get_daterange_results(
+        self, name_list: list[str]
+    ) -> dict[str, datetime.datetime]:
+        """
+        Use a list of "expected results" to determine the earliest and the
+        latest creation date of the set of datasets.
+
+        Args:
+            name_list: List of dataset names
+
+        Returns:
+            {"from": first_date, "to": last_date}
+        """
+        from_time = datetime.datetime.now(datetime.timezone.utc)
+        to_time = datetime.datetime(
+            year=1970, month=1, day=1, tzinfo=datetime.timezone.utc
+        )
```

Comment:
You're sure that "now" is far enough in the future?  I would go with something like:
```python
        from_time = datetime.datetime.fromtimestamp(
            9999999999, tz=datetime.timezone.utc
	)
        to_time = datetime.datetimefromtimestamp(0, tz=datetime.timezone.utc)
```

Reply:
I literally lifted this helper from the old `test_datasets_daterange.py` and the only change was adding the `_daterange` in the name since we already had a `get_results`. I don't see any point in worrying about the implementation now. If this were production code, yeah, "now" isn't late enough to catch time travelling performance engineers, but for our unit test environment there's no advantage and it's not worth the churn.